app/component/SaveCustomizedSettingsButton.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import Snackbar from 'material-ui/Snackbar';
+
+import { FormattedMessage } from 'react-intl';
+import { locationShape } from 'react-router';
+import { setCustomizedSettings } from '../store/localStorage';
+
+class SaveCustomizedSettingsButton extends React.Component {
+  static contextTypes = {
+    location: locationShape.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      autoHideDuration: 940,
+      message: 'settings-saved',
+      open: false,
+    };
+  }
+
+  setSettingsData = () => {
+    // Test if has new set values
+    const settings = {
+      accessibilityOption: this.context.location.query.accessibilityOption
+        ? this.context.location.query.accessibilityOption
```

Comment:
I wonder if the valid this.context.location.query.accessibilityOption=0 value cause problems here. Will it overwrite previously set  accessibilityOption=1 value?